src/contentmanager.cpp

```diff
@@ -214,12 +217,13 @@ QString ContentManager::downloadBook(const QString &id)
     return QString::fromStdString(download->getDid());
 }
 
-void ContentManager::eraseBookFilesFromComputer(const QString fileToRemove)
+void ContentManager::eraseBookFilesFromComputer(const QString bookPath)
```

Comment:
I think this method is a bit too complex.
We only use this method in eraseBook and cancelBook. There, we create a Qstring from the book path and then we split it in the function to get the different parts.

We should change the signature of the method to directly give the different information we need.